releasenotes/releasenotes.rst

```diff
@@ -27,11 +27,12 @@ The default FHIR version of the `public Firely Server endpoint <https://server.f
 .. _vonk_releasenotes_490:
 
 Release 4.9.0, June 27th, 2022
------------------------------
+------------------------------
 
 Feature
 ^^^^^^^
 
+#. We introduced a specific endpoint for healthchecks, see :ref:`feature_healthcheck``.
```

Comment:
This should probably go into new section for 4.5.0. Let's update this as soon as the releasenotes are merged into the current develop branch.